### ISSUE TITLE
Shortcut for Insert To-do not working

### ISSUE THREAD

**Issue by ShintaroOkuda** [HappenApps/Quiver#1424]
`CTRL+SHIFT+t` (Insert To-do) is not working.

`CTRL+SHIFT+r` (Insert Horizontal Rule) is working.

**Comment by macdevign** [comment]
Are you using (Version 3.2.7 (201)) of quiver ?

As both keys are working fine for me in Text Cell.

**Comment by ShintaroOkuda** [comment]
Yes, I am using Version 3.2.7 (201) and CTRL+SHIFT+t (Insert To-do) is not working.
